lib/event_serializer/config.ex

```diff
@@ -3,12 +3,12 @@ defmodule EventSerializer.Config do
   Helpers for getting config
   """
 
-  def topic_name do
-    :event_serializer |> EnvConfig.get(:topic_name) |> not_nil_topic_name
+  def topic_names do
+    :event_serializer |> EnvConfig.get(:topic_names) |> not_nil_topic_names
```

Comment:
I think we need to makea change here to receive an array https://github.com/quiqupltd/event_serializer/blob/master/config/config.exs#L18